app/controllers/drops_controller.rb

```diff
@@ -69,8 +69,8 @@ def upvote
   private
 
     def set_drop
-      @drop = Drop.find(params[:id].to_i)
-    end
+      page_not_found unless @drop = Drop.find_by(id: params[:id].to_i)
```

Comment:
I'm not sure if you can do this. I think instead it should be:

`@drop = Drop.find_by(id: params[:id].to_i) unless page_not_found` 

that being said I'm not totally 100% sure what `page_not_found` comes from.... but give that a shot, & if it doesn't work, try writing this a different way. You can play around w/ binding.pry too & see what happens here.